File: example_code/javascript/voice_generation.js
function GenerateVoice(signature) {
	const xhr = new XMLHttpRequest();
	const audio = new Audio();
	const api_url = "https://api.prosody-tts.com/api/ttsapi/voice-generation/" + signature + "/generate/"
	let url = null;

	xhr.open("GET", api_url, true);
    xhr.setRequestHeader("Authorization", "Api-Key {홈페이지에서 발급받은 APIKey}")
	xhr.addEventListener("load", function () {
		url = URL.createObjectURL(xhr.response);
		audio.src = url;
		audio.play();
	});
	xhr.send();
	audio.addEventListener('ended', function() {
		URL.revokeObjectURL(url);
	}, false);
}

File: example_code/c#/voice_generation.cs
var url = "https://api.prosody-tts.com/api/ttsapi/voice-generation/{signature}/generate/"

var client = new RestClient(url);
client.Timeout = -1;
var request = new RestRequest(Method.GET);
request.AddHeader("Authorization", "Api-Key {홈페이지에서 발급받은 APIKey}")
IRestResponse response = client.Execute(request);
